src/components/Brushbox.tsx

```diff
@@ -35,10 +47,15 @@ const BrushBox = ({setBrushSelection}) => {
     )
   })
 
+  const selectDelete = clicked === -1 ? "selected delete" : "";
```

Comment:
lets move off of "clicked" and using arbitrary numbers to represent it. This is what people would call "magic numbers" and is an antipattern in coding. Can we determine the state of what is clicked by what is in `brushSelection` alone?